web-api/src/persistence/postgres/acquireOneDbConnection.ts

```diff
@@ -0,0 +1,11 @@
+import { ConnectionBuilder } from 'kysely';
```

Comment:
This file:

"Hey, I need a particular connection from the connection pool."

Why might you want this?

- A `pg_advisory_lock` needs to be opened/closed on the same connection, so trying to call pool.connect when setting a lock and then again on releasing a lock could cause an issue since, in theory, you could get one connection the first time and another the second.
- Because transactions are "one per connection," a different or further implementation of transactions might want to keep track of a specific transaction. (In this PR, however, it is handled by kysely.transacation.)
- Etc.

Reply:
TODO: Will this actually persist the connection after a query is executed?

Reply:
Sadly, as I learn more about this, I think this is actually not quite right.